components/breadcrumb/src/Crumb.js

```diff
@@ -14,38 +14,39 @@ import { SrOnly } from '../../accessibility-helpers/src';
  * Breadcrumb: Breadcrumbs are styled navigational links used to indicate a user journey or path. They are a simple, effective and proven method to aid orientation.
  */
 export const Crumb = ({ children, last, ...props }) => {
```

Comment:
should probably set a proptype for last

`last: PropType.Bool,`

@dominikwilkowski We proptype check everything right? Even these type of props (used internally for styling purposes)